xml/decoders/lenz/motorControl_cv50.xml

```diff
@@ -91,66 +103,77 @@
 	<comment xml:lang="it">Valori da 0 a 5</comment>
 	<label xml:lang="de">Motortyp (0-5)</label>
 	<comment xml:lang="de">Wertebereich 0-5</comment>
-        <label xml:lang="cs">Typ motoru</label>
-        <comment xml:lang="cs">Meze 0-5</comment>
+    <label xml:lang="cs">Typ motoru</label>
+    <comment xml:lang="cs">Meze 0-5</comment>
+    <label xml:lang="ca">Tipus de motor</label>
+    <comment xml:lang="ca">Rang 0-5</comment>
   </variable>
   <variable CV="50" mask="XXVXXXXX" item="EMF Switch" default="0">
 	<enumVal>
 	  <enumChoice choice="Inactive">
 		<choice>Inactive</choice>
 		<choice xml:lang="it">Inattivo</choice>
 		<choice xml:lang="de">Aus</choice>
-                <choice xml:lang="cs">Neaktivní</choice>
+        <choice xml:lang="cs">Neaktivní</choice>
+        <choice xml:lang="ca">Inactiu</choice>
 	  </enumChoice>
 	  <enumChoice choice="Active">
 		<choice>Active</choice>
 		<choice xml:lang="it">Attivo</choice>
 		<choice xml:lang="de">Ein</choice>
-            <choice xml:lang="cs">Aktivní</choice>
+        <choice xml:lang="cs">Aktivní</choice>
+        <choice xml:lang="ca">Actiu</choice>
 	  </enumChoice>
 	</enumVal>
 	<label>EMF Switch</label>
 	<label xml:lang="it">Interruttore EMF</label>
 	<label xml:lang="de">EMK Teiler</label>
-        <label xml:lang="cs">Vypínač EMF</label>
+    <label xml:lang="cs">Vypínač EMF</label>
+    <choice xml:lang="ca">Control EMF</choice>
```

Comment:
This should probably be a `<label>` element, not a `<choice>` element.  The `<choice>` here is failing validation.